bitnami/postgresql/Chart.yaml

```
@@ -26,4 +26,4 @@ name: postgresql
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/postgresql
   - https://www.postgresql.org/
-version: 11.6.25
+version: 11.6.26
```

Since this is implementing a new parameter, could you increase the minor instead?

```suggestion
version: 11.7.0
```

---

Thanks